common/beerocks/hostapd/include/configuration.h

```diff
@@ -68,14 +68,24 @@ class Configuration {
      * @brief set key/value for the given vap
      * @details set the key/vale for the given vap, either replace or create.
      * @param 
-     * - the vap to set values for (string)
+     * - the vap to set the value for (string)
      * - the key to set its value (string)
      * - the value (int)
      * @return 
      * true - the vap exists, values were set.
      * false - the given vap was not found
      */
     bool set_create_vap_value(const std::string &vap, const std::string &key, const int value);
+
+    /**
+     * @brief get the value of the given key for the given vap
+     * @param 
```

Comment:
Is this a valid doxygen format for `@param`s?

Reply:
Nope.